support-lambdas/acquisition-events-api/README.md

```diff
@@ -0,0 +1,10 @@
+# Acquisition Events Api
+- API for sending in-app acquisitions(subscriptions) to BigQuery using a lambda
+
+## Testing
+To test locally:
+- Uncomment <code>@Ignore</code> from <code>LambdaSpec.scala</code>
```

Comment:
"Comment out..."

Reply:
also, you can use ` instead of <code> tags

Reply:
👍